Resources/Prototypes/Entities/Effects/exclamation.yml

```diff
@@ -0,0 +1,30 @@
+- type: entity
+  id: Exclamation
+  name: exclamation
+  noSpawn: true
+  save: false
```

Comment:
Why is save set to false?

Reply:
This prototype used for cardboardBox so idk. If you think that this is important then i change it

Reply:
It's not needed, if you're putting stuff in yaml you should have some idea of what each bit is doing before adding it.